docs/modules/ROOT/pages/erc1155-metadata-uri.adoc

```diff
@@ -0,0 +1,70 @@
+= ERC-1155 Metadata URI
+
+The OpenZeppelin xref:erc1155.adoc[ERC-1155] Metadata URI extension is needed to manage and store URIs for individual tokens. This extension allows each token to have its own unique URI,
+which points to a JSON file that conforms to the https://eips.ethereum.org/EIPS/eip-1155#erc-1155-metadata-uri-json-schema[ERC-1155 Metadata URI JSON Schema].
+This is particularly useful for non-fungible tokens (NFTs) where each token is unique and may have different metadata.
+
+[[usage]]
+== Usage
+
+In order to make an xref:erc1155.adoc[ERC-1155] token with https://docs.rs/openzeppelin-stylus/0.2.0-alpha/openzeppelin_stylus/token/erc1155/extensions/metadata_uri/index.html[Metadata URI] flavour,
+you need to add the following code to your contract:
+
+[source,rust]
+----
+use openzeppelin_stylus::token::erc1155::{
+    extensions::Erc1155MetadataUri, Erc1155,
+};
+
+sol_storage! {
+    #[entrypoint]
+    struct Erc1155Example {
+        #[borrow]
+        Erc1155 erc1155;
+        #[borrow]
+        Erc1155MetadataUri metadata_uri;
+    }
+}
+
+#[public]
+#[inherit(Erc1155, Erc1155MetadataUri)]
+impl Erc1155Example {
+    pub fn mint(
```

Comment:
I believe we do not need to have `mint` and `mint_batch` fns in docs for Metadata URI extension. I would say that it should be enough:
```rust
#[public]
#[inherit(Erc1155, Erc1155MetadataUri)]
impl Erc1155Example {
    // ...
}
```
Like for [Erc20 Metadata](https://github.com/OpenZeppelin/rust-contracts-stylus/blob/main/docs/modules/ROOT/pages/erc20-metadata.adoc).